### PR TITLE
Vagrantfile

### LINKED ISSUE #8: Vagrantfile needs to be created
We need to add a Vagrantfile to this project to launch a development VM.

### PR DESCRIPTION
**Pull request by Ofekmn**
Resolves #8

### CODE REVIEW

**Comment by talko611** [comment]
Hi Ofek, @Ofekmn 
I think you can add the HTTP connection to the VM. other than that looks fine to me.

**Comment by Ofekmn** [comment]
> Hi Ofek, @Ofekmn I think you can add the HTTP connection to the VM. other than that looks fine to me.

Done.

**Comment by talko611** [comment]
Looks great from my perspective!

**Comment by talko611** [comment]
One last thing. In my opinion, it would be cleaner to squash all commits into 1

**Comment by kobihk** [comment]
> One last thing. In my opinion, it would be cleaner to squash all commits into 1

@talko611 you don't have to do it in this phase because we can do it once we merge the PR we have the option to do "squash and merge"

**Review by SaarOrus — APPROVED** [review]
LGTM

**Review by kobihk — CHANGES_REQUESTED** [review]
@Ofekmn Please create an issue about this PR
then add a description to this PR as you can see in: 
https://github.com/redhat-beyond/team2/pull/2
but please use the supported keywords as you can see in:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
in this way, you make sure that when PR will merge, the relevant issue will close automatically.

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant
```
[the remark this answers is not in the document]
Comment (Ofekmn) [reply .gitignore]: Done.

**Review by rotem2022 — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**